File: CS/13_Conversion/ToXlsxOptions.cs
using Spire.Pdf;
using Spire.Pdf.Conversion;
using System;
using System.Windows.Forms;


namespace ToXlsxOptions
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Load a PDF document from the specified file path.
            PdfDocument document = new PdfDocument();
            document.LoadFromFile(@"../../../../../../Data/ToXlsxOptions.pdf");

            // Create an XlsxLineLayoutOptions object with four parameters: convertToMultipleSheet, showRotatedText, splitCell, wrapText.
            // These parameters determine various options for converting PDF to Excel.
            XlsxLineLayoutOptions options = new XlsxLineLayoutOptions(false, false, false, false);

            // Set the PDF to Excel conversion options of the document.
            document.ConvertOptions.SetPdfToXlsxOptions(options);

            // Specify the output file name for saving as Excel.
            string output = "ToXlsxOptions.xlsx";

            // Save the document as Excel with the specified output file name and format (XLSX).
            document.SaveToFile(output, FileFormat.XLSX);

            //Launch the Pdf file
            PDFDocumentViewer(output);
        }

        private void PDFDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

    }
}

File: VB/13_Conversion/ToXlsxOptions.vb
Imports Spire.Pdf
Imports Spire.Pdf.Conversion

Namespace ToXlsxOptions
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new instance of PdfDocument
			Dim document As New PdfDocument()

			' Load the PDF document from the specified file path
			document.LoadFromFile("../../../../../../Data/ToXlsxOptions.pdf")

			' Specify the options for converting PDF to Excel
			' These four parameters represent: convertToMultipleSheet, showRotatedText, splitCell, wrapText
			Dim options As New XlsxLineLayoutOptions(False, False, False, False)

			' Set the PDF to Excel conversion options
			document.ConvertOptions.SetPdfToXlsxOptions(options)

			' Specify the output file path for the Excel file
			Dim output As String = "ToXlsxOptions.xlsx"

			' Save the document as an Excel file
			document.SaveToFile(output, FileFormat.XLSX)

			' Close the PDF document
			document.Close()

			' Launch the Pdf file
			PDFDocumentViewer(output)
		End Sub

		Private Sub PDFDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

	End Class
End Namespace
